src/backend/CassandraBackend.cpp

```diff
@@ -1540,13 +1652,39 @@ CassandraBackend::open(bool readOnly)
         if (!selectNFT_.prepareStatement(query, session_.get()))
             continue;
 
+        query.str("");
+        query << "SELECT DISTINCT token_id FROM " << tablePrefix << "nf_tokens";
+        if (!selectAllNFTIDs_.prepareStatement(query, session_.get()))
+            continue;
```

Comment:
I think this query needs to be paginated. This could end up being a lot of token ids, and might be too large of a result set.